integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonAnalysisRules.scala

```diff
@@ -250,13 +251,19 @@ case class CarbonPreInsertionCasts(sparkSession: SparkSession) extends Rule[Logi
 
       case p@InsertIntoTable(relation: LogicalRelation, _, child, _, _)
         if relation.relation.isInstanceOf[CarbonDatasourceHadoopRelation] =>
-        castChildOutput(p, relation, child)
+        var isMultipleInserts = false
```

Comment:
```suggestion
        var containsMultipleInserts = false
```
Please in all places

Reply:
done